timetracker/timetracker/urls.py

```diff
@@ -18,11 +18,35 @@
 from django.http import HttpResponse
 
 
-def hello_world(request):
-    return HttpResponse('Hello World')
+def hello_world(request, name):
+   name = request.GET.get('name')
```

Comment:
@gazwald in this case we didn't need to add `name` to the URL patterns (since we're just accessing the GET parameter).

What you're doing here is actually overriding the value from the URL anyway. Close. But not quite what we were looking for here.